addons/im_livechat/__manifest__.py

```diff
@@ -26,8 +26,7 @@
         "views/im_livechat_channel_views.xml",
         "views/im_livechat_channel_templates.xml",
         "report/im_livechat_report_channel_views.xml",
-        "report/im_livechat_report_operator_views.xml",
-        "data/im_livechat_channel_data.xml"
+        "report/im_livechat_report_operator_views.xml"
```

Comment:
You removed im_livechat_channel_data import ? You modified the file but also removed its import it seems.